src/pages/MetricDetail.svelte

```diff
@@ -260,6 +277,10 @@
     @include text-title-xs;
   }
 
+  .tags-container {
+    margin-top: -12px;
+    padding-bottom: 16px;
+  }
```

Comment:
We should space the tags out just a tiny bit, right now it looks a little bit cluttered. This can be done in a follow up though

Reply:
Yeah definitely. I fixed this up.